resources/js/Pages/Adminland/CompanyPTOPolicy/Index.vue

```diff
@@ -177,37 +177,7 @@ td, th {
                       <thead>
                         <tr class="f6 tc">
                           <th scope="col">{{ $t('account.pto_policies_month') }}</th>
-                          <th scope="col">1</th>
-                          <th scope="col">2</th>
-                          <th scope="col">3</th>
-                          <th scope="col">4</th>
-                          <th scope="col">5</th>
-                          <th scope="col">6</th>
-                          <th scope="col">7</th>
-                          <th scope="col">8</th>
-                          <th scope="col">9</th>
-                          <th scope="col">10</th>
-                          <th scope="col">11</th>
-                          <th scope="col">12</th>
-                          <th scope="col">13</th>
-                          <th scope="col">14</th>
-                          <th scope="col">15</th>
-                          <th scope="col">16</th>
-                          <th scope="col">17</th>
-                          <th scope="col">18</th>
-                          <th scope="col">19</th>
-                          <th scope="col">20</th>
-                          <th scope="col">21</th>
-                          <th scope="col">22</th>
-                          <th scope="col">23</th>
-                          <th scope="col">24</th>
-                          <th scope="col">25</th>
-                          <th scope="col">26</th>
-                          <th scope="col">27</th>
-                          <th scope="col">28</th>
-                          <th scope="col">29</th>
-                          <th scope="col">30</th>
-                          <th scope="col">31</th>
+                          <th v-for="n in 31" :key="n" scope="col">{{ n }}</th>
```

Comment:
@djaiss This is a simplification

Reply:
@asbiin omg I knew we have a simpler way. I don't know JS well enough to think about that.